connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectClusterAssertions.java

```diff
@@ -455,48 +455,42 @@ public void assertConnectorIsFailedAndTasksHaveFailed(String connectorName, int
     }
 
     /**
-     * Assert that a connector and its tasks are not running.
+     * Assert that a connector does not exist. This can be used to verify that a connector has been successfully deleted.
      *
      * @param connectorName the connector name
      * @param detailMessage the assertion message
      * @throws InterruptedException
      */
-    public void assertConnectorAndTasksAreNotRunning(String connectorName, String detailMessage)
+    public void assertConnectorDoesNotExist(String connectorName, String detailMessage)
             throws InterruptedException {
         try {
             waitForCondition(
-                () -> checkConnectorAndTasksAreNotRunning(connectorName),
+                () -> checkConnectorDoesNotExist(connectorName),
                 CONNECTOR_SETUP_DURATION_MS,
-                "At least the connector or one of its tasks is still running");
+                "The connector should not exist.");
```

Comment:
This condition details should be the opposite i.e `The Connector exists` or something like that, since it's used to construct the message when the assert failed even after waiting for the timeout. The older message also seems to suggest the same

Reply:
The condition details ends up being used here - https://github.com/apache/kafka/blob/7872a1ff5b2e9a0fbbe3d71180a97e29f1549d4f/clients/src/test/java/org/apache/kafka/test/TestUtils.java#L331-L333

So the older message should actually have been inverted.